app/Domain/Newsletter/Controllers/NewsletterController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Newsletter\Controllers;
+
+use App\Http\Controllers\Controller;
+use Mailjet\Client;
+use Mailjet\Resources;
+use Mailjet\Response;
+
+final class NewsletterController extends Controller
+{
+    public function handle(string $email): Response
+    {
+        $mailingListId = config('newsletter.id');
+        $apiKey = config('newsletter.apiKey');
+        $apiSecret = config('newsletter.apiSecret');
+
+        $mailJetClient = new Client($apiKey, $apiSecret);
+        return $mailJetClient->post(
+            Resources::$ContactslistManagecontact,
```

Comment:
This is weird 😅

Reply:
I just followed the Mailjet Docs. I guess there is little we can do about it.

Reply:
Indeed we can't change that. I should have commented and already resolved the conversation 🙂